File: Frontend/src/app/transportObjects/CodeToLombokizeTO.ts
import {AnnotationsConfig} from "./AnnotationsConfig";

export class CodeToLombokizeTO {

    codeToLombokize: string;
    annotationsConfig: AnnotationsConfig;


    constructor(codeToLombokize: string, annotationsConfig: AnnotationsConfig) {
        this.codeToLombokize = codeToLombokize;
        this.annotationsConfig = annotationsConfig;
    }
}

File: Backend/src/main/java/com/simmondobber/lomboker/lombokize/transportObjects/CodeToLombokizeTO.java
package com.simmondobber.lomboker.lombokize.transportObjects;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
public class CodeToLombokizeTO {

    private String codeToLombokize;
    private AnnotationsConfig AnnotationsConfig;
}
